src/common/m_constants.fpp

```diff
@@ -26,4 +26,11 @@ module m_constants
     real(kind(0d0)), parameter :: acoustic_spatial_support_width = 2.5d0 !< Spatial support width of acoustic source, used in s_source_spatial
     real(kind(0d0)), parameter :: dflt_vcfl_dt = 100d0 !< value of vcfl_dt when viscosity is off for computing adaptive timestep size
 
+    !< Broadband acoustic source constant (reference: Tam et al. JSV 2005)
+    integer, parameter :: num_broadband_freq = 100 !< The number of sine wave frequencies in broadband acoustic source.
+    real(kind(0d0)), parameter :: broadband_freq_lowest = 500d0 !< The lower sine wave frequency bound in broadband acoustic source.
+    real(kind(0d0)), parameter :: broadband_bandwidth = 100d0 !< The bandwidth of the discretized sine wave frequencies.
```

Comment:
it seems like the user should be able to set of a couple of these parameters themself, like the bandwidth itself, right? or is that something that you just set and would never really want to change?

Reply:
It's something user can set as for their own frequency range of interest. I can make all the constant here as input parameters.

Reply:
Does it seem like the frequency range would be something the user would want to set? It seems this way to me. If so, please make it a user input instead of a hard-coded constant. Thanks!

Reply:
I've added the requested input parameters and refactored the code.